sdk/android/skin/src/main/java/com/ooyala/android/skin/OoyalaSkinLayoutController.java

```diff
@@ -402,6 +448,28 @@ public void setFullscreenButtonShowing(boolean showing) {
 
   }
 
+  @Override
+  public void switchVRMode(VrMode vrMode) {
+    Context context = getLayout().getContext();
+    if (context instanceof Activity) {
```

Comment:
same comment as above regarding the context being an Activity

Reply:
Also added a `DebugMode.logE(TAG, "Trying to switch VR mode. The context isn't an instance of Activity.");` if it's not.